docs/source/tutorials/raptor.ipynb

```diff
@@ -0,0 +1,242 @@
+{
```

Comment:
<p>"The default RAPTOR Worker has a <code>test</code> function defined which we can use for proof of concept:" -&gt; The default RAPTOR Worker has a built-in <code>test</code> function which you can use for testing purpose:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/radical-cybertools/radical.pilot/pull/2883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
If you agree and address the above, I think it is ready to be merged. Thank you!

Reply:
Done!